src/wechaty.ts

```diff
@@ -683,27 +683,24 @@ export class Wechaty extends Accessory implements Sayable {
             const inviter = this.Contact.load(inviterId)
             await inviter.ready()
 
-            this.emit('room-join', room, inviteeList, inviter)
-            room.emit('join', inviteeList, inviter)
+            this.emit('room-join', room, inviteeList, inviter, new Date(timestamp))
```

Comment:
Create a `date` var to store the `new Date()` value. 

Then you can use it for twice.